blocks/transformations/no_lemmatizer.py

```diff
@@ -0,0 +1,35 @@
+from .base import Transformation
+from configs.constants import Const
+import pandas as pd
+from utils.spacy import get_spacy
+from typing import List, Any
+import spacy
+from type import DataType
+
+
+class NoLemmatizer(Transformation):
```

Comment:
so, unfortunately we need a specific step after the spacy tokenizer no matter what!
right now, it's either Lemmatizer or NoLemmatizer.
This sounds stupid. maybe the spacy tokenizer should encompass lemmatization (if we want it) and stop word removal (if we want it)?

Reply:
We could add it as a seperate preprocess section of each of these models? Or just simply add it to the model as you suggested.

Reply:
can do both! what do you mean by separate preprocess section?